comp/prizes/acquire-prizes-edible.yaml

```diff
@@ -0,0 +1,24 @@
+summary: Organise edible prizes for the competition
+
+priority: should
+
+component: Competition
+
+milestone: $SRYYYY Competition
+
+area-owner: production
+
+description: >-
+  The prizes for the competition are a combination of trophies and edible prizes.
+  This task only considers the edible prizes; there is a separate task for trophies.
```

Comment:
Not sure if this is easy to do, but if possible, it would be good to link to the trophies task here.